openmeetings-db/src/main/java/org/apache/openmeetings/db/dao/basic/ConfigurationDao.java

```diff
@@ -571,8 +572,12 @@ private void addCspRule(CSPHeaderConfiguration cspConfig, CSPDirective key, Stri
 	}
 
 	public void updateCsp() {
-		setGaCode(getString(CONFIG_GOOGLE_ANALYTICS_CODE, null));
+		if (!getBool(CONFIG_CSP_ENABLED, true)) {
+			WebApplication.get().getCspSettings().blocking().disabled();
+			return;
+		}
 
+		setGaCode(getString(CONFIG_GOOGLE_ANALYTICS_CODE, null));
```

Comment:
this call should go **before** `CONFIG_CSP_ENABLED` check

Reply:
fixed